src/nci/NfcManager.cpp

```diff
@@ -184,8 +191,12 @@ bool NfcManager::initialize()
     }
   }
 
-  if (sIsNfaEnabled)
+  if (sIsNfaEnabled) {
+#ifdef NFCC_PN547
+    EXTNS_Close();
+#endif
```

Comment:
I saw many #ifdef occurrences
perhaps you could add another function to wrap this.

void externalClose() {
  #ifdef NFCC_PN547
    EXTNS_Close();
 #else 
   ....
#endif  
}